sale_delivery_split_date/tests/test_sale_delivery.py

```diff
@@ -45,6 +45,10 @@ def test_check_single_date(self):
         self.assertEqual(
             len(self.so.picking_ids), 1,
             "There must be 1 picking for the SO when confirmed")
+        for picking in self.so.picking_ids:
+            self.assertEqual(
+                picking.sale_id, self.so,
+                "The sale order form picking must be the confirmed SO")
```

Comment:
could you rephrase it a bit? what's `form` stand for?

Reply:
misspelled, `from`